text/0021-internal-data-monitoring.md

```diff
@@ -0,0 +1,109 @@
+---
+rfc pr: https://github.com/UCLA-Creative-Labs/cl-rfcs/pull/xxx
+tracking issue: https://github.com/UCLA-Creative-Labs/cl-rfcs/issues/21
+---
+
+# [Watchdog]
+
+`Watchdog` is a monorepo that will contain a variety of Creative Labs data analysis tools for internal use. 
+
+## Blurb
+
+The goal of this project is to allow all the teams across Creative Labs board to make data-driven decisions. We want to set up a system where all members can access previous data and use insights from the past to make good judgement calls in the present.
+
+Watchdog is meant to be a monorepo to house various data analysis tools for Creative Labs. There are two main projects to implement currently. The first is a continuation of the current Watchdog project, which is a tool to get data analytics on Creative Labs projects. Currently, it fetches project application information and displays its various statistics. We want to organize this by creating user datasets that would keep track of the same applicant through multiple application cycles. An improvement upon the original Watchdog project would be to allow board members to better access the data - through a web app that allows users to choose their own method of displaying data (bar graph, pie chart, etc) or choosing a specific time period with various filters.
+
+The second project would be a program that monitors how others interact with Creative Labs. This would include where (and at what frequency) users are navigating towards on our main website or understanding the correlation between social media posts interactions (ads, likes, shares) and event attendance. This project is primarily meant for Creative Labs board members but we could look at displaying some of the data publicly or publishing our findings through a blog post for marketing content!
```

Comment:
I would like it if both of these projects were separated into sections with a heading for each of them!